tox.ini

```diff
@@ -111,11 +111,10 @@ deps = {[testenv]basedeps}
 commands =
     nosetests \
       --with-gae \
+      --gae-lib-root={env:GAE_PYTHONPATH:google_appengine} \
```

Comment:
@jonparrott I had `/usr/local/google_appengine` symlink-ed to `${HOME}/google-cloud-sdk/platform/google_appengine` on one machine and not at all on another. Since that is the [default][1], I got spurious local success that failed on Travis (and on my other machine).

[1]: https://github.com/Trii/NoseGAE/blob/b7c91f8c45d431b7b9fdcca33f40546a2bc12fe0/nosegae.py#L37